runners/spark/src/main/java/org/apache/beam/runners/spark/SparkPipelineOptions.java

```diff
@@ -34,18 +34,6 @@
  */
 public interface SparkPipelineOptions extends SparkCommonPipelineOptions {
 
-  @Description("Set it to true if event logs should be saved to Spark History Server directory")
```

Comment:
Historically we avoid at all costs to introduce variables that are already resolved by Spark Configuration, so these variables should come from the Spark configuration to avoid duplicated non synchronized configuration changes.